src/core/Order.js

```diff
@@ -0,0 +1,39 @@
+/*jshint bitwise: false*/
+
+getJasmineRequireObj().Order = function() {
+  function Order(options) {
+    this.random = 'random' in options ? options.random : true;
+    var seed = this.seed = options.seed || generateSeed();
+    this.sort = this.random ? randomOrder : naturalOrder;
+
+    function naturalOrder(items) {
+      return items;
+    }
+
+    function randomOrder(items) {
+      items.sort(function(item) {
```

Comment:
Ideally, this should just return the new sorted array and not change the order of the existing array.